README.md

```diff
@@ -25,10 +25,17 @@ cd mailer
 sbt run
 ```
 
+### Running `analyzer`
+
+``` Shell
+cd analyzer
+sbt run
+```
+
 ### Testing with simple producer
 
 ``` Shell
-docker exec -it kafka /opt/kafka_2.12-0.10.2.1/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic newspaper
+docker exec -it kafka /opt/kafka_2.12-0.10.2.1/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic newspaper-content
```

Comment:
That's a good point - do we want to stream all events via the same topic or different? Guess we don't need any happens-before correlation, so we could save us from long rewinding if we split the topics ... (Just a random question atm, we can discuss in more details on Slack)